jena-base/src/main/java/org/apache/jena/atlas/lib/StrUtils.java

```diff
@@ -33,32 +35,34 @@ private StrUtils() {}
     
     /** strjoin with a newline as the separator */
     public static String strjoinNL(String... args) {
-        return join("\n", args);
+        return String.join("\n", args);
     }
 
     /** strjoin with a newline as the separator */
     public static String strjoinNL(List<String> args) {
-        return join("\n", args);
+        return String.join("\n", args);
     }
 
-    /** Concatentate strings, using a separator */
+    /** 
+     * Concatentate strings, using a separator
+     * 
+     * @deprecated Prefer String.join(sep, args)
+     */
+    @Deprecated
```

Comment:
If we are going to deprecate this, then I really would like to see the changes propagated through the codebase, not left pending.

Currently, at least for me, Jena is warning free in the main modules so I can see when warnings appear. Having unfinished deprecations means that it is harder to notice warnings appearing.